pkg/config/config.go

```diff
@@ -227,17 +228,27 @@ func parseServiceYAML2_1(serviceYAML *service2_1) (*Service, error) {
 	service.Environment = make(map[string]string, len(serviceYAML.Environment.Values))
 	for _, pair := range serviceYAML.Environment.Values {
 		var value string
+		if len(pair.Name) == 0 {
```

Comment:
Can probably just check pair.Name == ""

Reply:
Yeah, we're doing this in a lot of places and linter doesn't complain. So prefer doing this for consistency.